packages/blockchain-wallet-v4-frontend/src/scenes/AuthorizeLogin/AuthorizeLogin.template.success.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react'
 import { FormattedMessage } from 'react-intl'
-import { Icon, Separator } from '@blockchain-com/constellation'
-import { IconAlert, IconCheckCircle, IconLaptop } from '@blockchain-com/icons'
+import {
+  IconAlert,
+  IconCheckCircle,
+  IconLaptop,
+  PaletteColors,
+  Separator
```

Comment:
Looks like separator isn't in constellation 0.1?

Reply:
It might have been added into constellation after I published 0.1.0. I'll fix.